src/jiradeps/jirawrapper.py

```diff
@@ -118,7 +121,8 @@ def get_team(issue: Issue) -> Optional[str]:
 def get_blocked_keys(issue: Issue) -> List[str]:
     links = issue.fields.issuelinks
     return [link.outwardIssue.id for link in links
-            if str(link.type) == 'Blocks' and hasattr(link, 'outwardIssue')]
+            if str(link.type) in BLOCKING_RELATIONS
+            and hasattr(link, 'outwardIssue')]
```

Comment:
if you bother to insert a line break please indent the second line by one level.

Reply:
I get a PEP8 error of I do that. Adding the line break is required for PEP8 compliance.